scripts/functions.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+SCRIPTS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd -P )"
+ROOT_DIR=$(dirname "$SCRIPTS_DIR")
+
+# Generates a stable version that will be used instead of reading 
+# from dynver. This is useful when running tasks that can change the
+# code for all the samples, for example, when running format tools.
+function echo_cf_version() {
```

Comment:
if there are issues with dynver, let's analyze them and agree on solutions, but this should not be done in this way.

Reply:
There is no issue with dynver: if the repo is dirty it will generate a version that contains the time.

Reply:
correct, and building/running examples should not result in a dirty repo

Reply:
Done. See https://github.com/lightbend/cloudflow/pull/1059/commits/2b356c499ba8328da59804f1a05f3f174b133337.

Reply:
nice 👍